vcfa/provider_test.go

```diff
@@ -126,14 +126,14 @@ func TestAccClientUserAgent(t *testing.T) {
 		InsecureFlag: testConfig.Provider.AllowInsecure,
 	}
 
-	vcdClient, err := clientConfig.Client()
+	vcfaClient, err := clientConfig.Client()
```

Comment:
Maybe we should replace all:

`vcfaClient -> tmClient`
`govcfaClient -> tmClient`
``

```suggestion
	tmClient, err := clientConfig.Client()
```

Reply:
Changed `vcfaClient`, but not occurrences of `govcfaClient` found

Reply:
Now it's `gotmClient`, but it's okay, not nitpicking anymore 😄